core/email_api.php

```diff
@@ -1216,8 +1216,13 @@ function email_send_all( $p_delete_on_failure = false ) {
 		}
 
 		if( !$t_email_sent ) {
-			if( $p_delete_on_failure ) {
-				email_queue_delete( $t_email_data->email_id );
+			# Delete emails that were submitted more than 7 days ago (MantisHub change)
```

Comment:
What is the point of the _MantisHub change_ comment ?

Reply:
Copy and paste error, since I originally implemented this as a MantisHub specific change without the config option.  Fixed.